### ISSUE TITLE
Make version SHACL shapes less restrictive

### ISSUE THREAD

**Issue by holycrab13** [dbpedia/databus#167]
SHACL validation should be the first thing to execute since it can also reliably detect malformed RDF.

**Comment by JJ-Author** [comment]
yeah we have to be a bit careful here. i think its best have a shacl library with all the tests and then one shape with optional values before autocompletion and one after autocompletion (to verify that this does not mess up sth)

**Comment by JJ-Author** [comment]
another (probably easier) idea by me discussed in todays meeting was the following
- add special triple to propertyshape e.g. sth like `databus:ApiAutocompletionStrategy "explain how it is autocompleted"`
- generate an automatically rewritten version of the shacl test that can be applied prior any processing in javasript by rewriting by overwriting the sh:mincount with value 0 for all that shapes having this autocompletion property

this way we can solve 2 issues for one: we can do easier documentation and can display it easily in the docu for every property how it will be autocompleted.
